src/DocumentFormat.OpenXml/Packaging/EmbeddedControlPersistenceBinaryDataPartExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace DocumentFormat.OpenXml.Packaging
+{
+    internal static partial class EmbeddedControlPersistenceBinaryDataPartExtensions
+    {
+        /// <summary>
+        /// Adds a EmbeddedControlPersistenceBinaryDataPart to a parent IAddExtensiblePart class.
```

Comment:
```suggestion
        /// Adds a EmbeddedControlPersistenceBinaryDataPart as a relation of the part. 
```

Reply:
I made some changes, let me know. I prefer ISupportExtensibleRelationship if that works for you.

Reply:
What's the purpose of extensible in the name?

Reply:
Changed to ISupportRelationship. The other was probably not what I was originally thinking.